### DISCUSSION TITLE
2단계 초기화

### DISCUSSION THREAD

**Discussion by iammiori** [swift-in-gangnam/swift-grammar#29] [Q&A] [answered]
제가 한글을 못하나요... 이거 읽어도 읽어도 이해가 안가요ㅠㅠ

**Comment by hyunjung-ann** [comment]
저도 규칙이 헷갈려서 뒤쪽 예제 풀 때 앞장이랑 계속 왔다갔다 했어요 ㅠㅠ 😥 초기화 단계 복잡하네여 ..

**Comment by Chuncheonian** [comment, marked as answer]
@iammiori 전 이거보고 이해했어요! https://ios-development.tistory.com/182
